modern-phytomorphology.csl

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Modern Phytomorphology</title>
+    <title-short>ModPhytomorphol</title-short>
+    <id>http://www.zotero.org/styles/modern-phytomorphology</id>
+    <link href="http://www.zotero.org/styles/modern-phytomorphology" rel="self"/>
+    <link href="hhttps://phytomorphology.org/journal/author-guidelines/" rel="documentation"/>
+    <author>
+      <name>Andrew Novikoff</name>
+      <email>novikoffav@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <issn>2226-3063</issn>
+    <eissn>2227-9555</eissn>
+    <updated>2016-10-01T04:26:55+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor-translator">
+    <names variable="editor translator" delimiter="; " prefix="(" suffix=")">
+      <name font-variant="small-caps" et-al-min="19" et-al-use-first="19" et-al-subsequent-min="19" et-al-subsequent-use-first="19" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
```

Comment:
@novikoffav, you can delete the `et-al-min="19" et-al-use-first="19" et-al-subsequent-min="19" et-al-subsequent-use-first="19"` here as well, since these values carry over from `<bibliography/>` (the only place where this macro is used).